### PR TITLE
[REG-1258] Fix docs related to new state fields, add new onboarding flow

### PR DESCRIPTION
**Pull request by vontell**
Adds the following:

* New instructions in the getting started doc related to the new onboarding flow
* New instructions in the getting started doc for the isPlayer and isRuntimeObject state flags
* Add isPlayer and isRuntimeObject information to the RGState docs

### CODE REVIEW

File: docs/studios/unity/tutorials/first_tutorial.md [hunk 1]
```diff
@@ -31,11 +31,8 @@ power ups (represented by blue spheres). We will cover:
 
 The first step is to create an account on Regression Games. Visit the [Regression Games site](https://play.regression.gg)
 and click "sign up". Enter a username, email, and password. Once you log 
-in, you will be prompted to connect a GitHub account. After connecting your account, the Regression Games platform will 
-be able to pull and live reload bots through GitHub repositories.
-
-You will also be prompted to enter a Minecraft username - you can put anything for this field (it will be removed
-soon as part of our new studio-centric experience).
+in, select "I am a game developer", you will be prompted to connect a GitHub account. After connecting your account,
```
Comment (RG-nAmKcAz) [inline docs/studios/unity/tutorials/first_tutorial.md:34]:
```suggestion
in, select "I am a game developer". You will be prompted to connect a GitHub account. After connecting your account, 
```

**Review by RG-nAmKcAz — APPROVED** [review]
1 small sentence structure tweak

File: docs/studios/unity/tutorials/first_tutorial.md (hunk 1 above)
Comment (vontell) [reply docs/studios/unity/tutorials/first_tutorial.md]: Committed!

File: docs/studios/unity/unity-sdk/RGState.mdx [hunk 2]
```diff
@@ -23,6 +23,8 @@ Name | Type  | Description
 `objectType` | `string` | A tag that informs the bot what kind of object this is, such as an `enemy` or `power_up`.  
 `syncPosition` | `boolean` | If true, the state relayed will include the position of the object.
 `syncRotation` | `boolean` | If true, the state relayed will include the rotation of the object.
+`isPlayer` | `boolean` | If true, this is an object that is controlled as a bot/player. Otherwise, it is a non-player controlled object in the scene.
```
Comment (abeizer) [inline docs/studios/unity/unity-sdk/RGState.mdx:26]: I'd like to see it clarified that "controlled" in this context means "controlled by Regression Games"

Comment (RG-nAmKcAz) [reply docs/studios/unity/unity-sdk/RGState.mdx:26]: That isn't necessarily true, isPlayer is used and intended to represent any player controlled object; whether by a bot or a human player.

**Review by rcornwall-reg — APPROVED** [review]

**Pull request merged**